hushline/templates/premium-waiting.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
```

Comment:
Nit. For readability / skimability, newlines between the `{% ... %}` blocks

Reply:
Fixed in f4c3e88573128e2960908e0dac6ee4d44c46e331